src/modules/shared/pages/innovation/data-sharing-and-support/data-sharing-and-support.component.ts

```diff
@@ -249,6 +254,22 @@ export class PageInnovationDataSharingAndSupportComponent extends CoreComponent
         }
       }
 
+      // Check if there are organisations to be suggested by the qualifying accessor
+      if (this.userType === UserRoleEnum.QUALIFYING_ACCESSOR) {
+        const userUnitId = this.authenticationStore.getUserContextInfo()?.organisationUnit?.id || '';
+
+        const engagingUnitsIds = results.innovationSupports
+          .filter(support => support.status === 'ENGAGING')
```

Comment:
We should use the Enum: `InnovationSupportStatusEnum`

Reply:
Yes!!